node/bft/examples/simple_node.rs

```diff
@@ -129,10 +129,10 @@ pub async fn start_bft(
         Arc::new(BFTMemoryService::new()),
         BatchHeader::<CurrentNetwork>::MAX_GC_ROUNDS as u64,
     );
-    // Initialize the gateway IP and dev mode.
-    let (ip, dev) = match peers.get(&node_id) {
-        Some(ip) => (Some(*ip), None),
-        None => (None, Some(node_id)),
+    // Initialize the gateway's storage mode.
+    let (ip, storage_mode) = match peers.get(&node_id) {
+        Some(ip) => (Some(*ip), StorageMode::Production),
```

Comment:
since this is an example, I wonder if the `Test` storage wouldn't be more fitting here; the `Development` mode might be a redundant option too